package.json

```diff
@@ -27,35 +27,22 @@
     "url": "git+https://github.com/syntest-framework/syntest-core.git"
   },
   "scripts": {
-    "build": "npm run build:compile",
-    "build:compile": "tsc --build",
-    "build:watch": "tsc --build --watch",
-    "clean": "rm -rf .nyc_output dist node_modules",
-    "clean:dist": "rm -rf dist",
-    "format": "prettier --config .prettierrc.json --ignore-path .prettierignore --write .",
-    "format:check": "prettier --config .prettierrc.json --ignore-path .prettierignore --check .",
-    "lint": "eslint --ignore-path .eslintignore .",
-    "lint:fix": "eslint --ignore-path .eslintignore . --fix",
-    "prepare": "npm run build",
-    "test": "mocha --config .mocharc.json",
-    "test:coverage": "nyc mocha --config .mocharc.json",
-    "test:watch": "mocha --config .mocharc.json --watch"
-  },
-  "dependencies": {
-    "@dagrejs/graphlib": "2.1.4",
-    "@syntest/cfg-core": "file:packages/cfg-core",
-    "bignumber.js": "9.0.1",
-    "chalk": "4.1.1",
-    "clear": "0.1.0",
-    "cli-progress": "3.11.2",
-    "fast-csv": "4.3.6",
-    "figlet": "1.5.2",
-    "globby": "^11.0.4",
-    "seedrandom": "3.0.5",
-    "shelljs": "0.8.5",
-    "winston": "3.8.2",
-    "yargs": "17.6.2"
+    "lerna:install": "npx lerna bootstrap",
+    "lerna:build:compile": "npx lerna exec npm run build:compile",
+    "lerna:build:watch": "npx lerna exec npm run build:watch",
+    "lerna:clean": "npx lerna exec npm run clean",
+    "lerna:clean:dist": "npx lerna exec npm run clean:dist",
+    "lerna:format": "npx lerna exec npm run format",
+    "lerna:format:check": "npx lerna exec npm run format:check",
+    "lerna:lint": "npx lerna exec npm run lint",
+    "lerna:lint:fix": "npx lerna exec npm run lint:fix",
+    "lint-staged": "lint-staged",
+    "lerna:setup-pre-commit-hooks": "husky install",
+    "lerna:test": "npx lerna exec npm run test",
+    "lerna:test:coverage": "npx lerna exec npm run test:coverage",
+    "lerna:test:watch": "npx lerna exec npm run test:watch"
```

Comment:
Do we want to make lerna hidden? we could still use the same old script names with just lerna in the middle? I am not sure what the convention here is.

Reply:
Not sure what you mean by hidden.
I also changed this file a bit.
Is this what you meant?

Reply:
In hindsight, let's just be completely transparent and show that we use Lerna